omnibus/files/private-chef-ctl-commands/chef-server-ctl

```diff
@@ -216,6 +216,17 @@ EOM
         "/admin/ping?wt=json"
       end
     end
+
+    # Override reconfigure to skip license checking
```

Comment:
@marcparadise you can indeed pass `--accept-license` but my understanding was that we didn't want that to happen for the Chef Server since it's fully open source.